ui/pages/swaps/select-quote-popover/select-quote-popover.stories.js

```diff
@@ -29,11 +27,16 @@ export default {
       options: ['Agg1', 'Agg2', 'Agg3', 'Agg4', 'Agg5', 'Agg6'],
     },
     quoteDataRows: {
-      control: 'object',
+      control: {
+        type: 'object',
+      },
     },
     hideEstimatedGasFee: {
       control: 'boolean',
     },
+    onSubmit: {
+      action: 'submit SelectQuotePopover',
```

Comment:
Our convention for actions is just to comment the prop
```suggestion
      action: 'onSubmit',
```